pkg/oc/cli/admin/createerrortemplate/create_error_template.go

```diff
@@ -43,7 +44,7 @@ func NewCommandCreateErrorTemplate(f kcmdutil.Factory, commandName string, fullN
 				kcmdutil.CheckErr(kcmdutil.UsageErrorf(cmd, err.Error()))
 			}
 
-			_, err := io.WriteString(out, errorpage.ErrorPageTemplateExample)
+			_, err := io.WriteString(streams.Out, errorpage.ErrorPageTemplateExample)
```

Comment:
Move this to a small Run method please, also `fmt.Fprintf(options.Out, errorpage.ErrorPageTemplateExample)` for consistency with other commands